tutorials/get-started-hadoop-spark/get-started-hadoop-spark.md

```diff
@@ -0,0 +1,222 @@
+---
+
+id: get-started-hadoop-spark
+summary: Learn how to operate a big data cluster, to analyse batch data with MapReduce or streaming data with Spark.
+categories: jaas
+tags: hadoop, spark, juju, jaas, big data, cloud
+difficulty: 2
+status: Published
+Published: 2017-06-01
+
+---
+
+# Get started with Hadoop Spark
+
+## Overview
+Duration: 1:00
+
+
+### About Hadoop Spark
+
+This provides a highly-available (HA) service on top of a cluster of machines, which is resistant to individual machine failure. It provides a flexible solution consisting of HDFS, MapReduce, and Spark that can process a wide variety of workloads.
+
+Hadoop is designed to scale to thousands of servers, each offering local computation and storage capacity, and is able to detect and handle failures at the application layer.
+
+[Learn more about this bundle](https://jujucharms.com/hadoop-spark/).
+
+![Hadoop Spark bundle](./images/hadoop-spark-bundle.png)
+
+
+### In this tutorial you’ll learn how to...
+
+- Get your Hadoop Spark cluster up and running, using [JAAS](https://jujucharms.com/jaas)
+- Operate your new cluster.
+- Create your first big data workload.
+- Change the execution mode of Spark in your cluster.
+
+
+### You will need...
+
+* An [Ubuntu One account](https://login.ubuntu.com/).
+* An SSH key in your JAAS account. If you haven’t done this already, [learn how to add your SSH key](https://jujucharms.com/docs/2.1/users-auth#credentials-and-ssh-keys).
+* Credentials for AWS, GCE or Azure
+
+
+
+
+## Deploying with JAAS
+Duration: 05:00
+
+To kick off, [open Hadoop Spark](https://jujucharms.com/new/?deploy-target=cs:bundle/hadoop-spark) and click the blue Deploy changes (56) button in the bottom right. Then, follow the steps to deploy your bundle.
+
+**Note:** *Even though the step is marked as optional, we suggest adding your public SSH key at this stage to allow you access to the machines provisioned for this model.*
+
+Deployment can take up to 30-45 minutes, as Juju creates new instances in the cloud and sets up the Hadoop Spark cluster components. Pending units are outlined in *orange*. Up and running are outlined in *black*.
+![Status GUI](./images/status-gui.png)
+
+Congratulations! You now have a Hadoop Spark cluster up and running.
+
+
+
+
+## Getting Started
```

Comment:
@caldav true - we'll use the original image for now and see to update it in the next iteration